yarn.lock

```diff
@@ -0,0 +1,4766 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
[minor] I didn't think we were switching to yarn in the SDK yet?

Reply:
I've been using yarn everywhere so far?  Happy to take this out though.